plugins/ingest/licenses/joni-LICENSE.txt

```diff
@@ -0,0 +1 @@
+Joni is released under the MIT License.
```

Comment:
I left these `LICENSE` and `NOTICE` files here as placeholders so the checker passes.

The only mention of JRuby's license is in the readme where they say "Joni is released under the MIT License".

I am not familiar with what we should place in the `NOTICE` files for projects that do not have their own official `NOTICE` file. is this mention enough?

and, since both joni and jcodings are projects with the same license under the JRuby umbrella, is one License and Notice file for JRuby sufficient to cover both projects?

@s1monw, do you have any recommendations for how to best deal with this?